internal/log/configure.go

```diff
@@ -6,8 +6,9 @@ import (
 
 // Config for the logger.
 type Config struct {
-	Level Level `help:"Log level." default:"info" env:"LOG_LEVEL"`
-	JSON  bool  `help:"Log in JSON format." env:"LOG_JSON"`
+	Level      Level `help:"Log level." default:"info" env:"LOG_LEVEL"`
+	JSON       bool  `help:"Log in JSON format." env:"LOG_JSON"`
+	Timestamps bool  `help:"Include relative timestamps in logs." env:"LOG_TIMESTAMPS"`
```

Comment:
It might be good to clarify that this is only in plain text logs, not JSON logs.